src/Sequencer4Module.cpp

```diff
@@ -134,9 +129,16 @@ Sequencer4Module::Sequencer4Module() {
     assert(seq4);
 }
 
+void Sequencer4Module::setModuleId(bool fromWidget) {
+    const auto seq4 = getSequencer();
+    if (seq4) {
+        seq4->undo->setModuleId(this->id, fromWidget);
+    }
+}
+
```

Comment:
This is clearer in Seq++, but this is the API that lets the UI widgets set the module ID, in case the audio thread isn't running.